pkg/features/features.go

```diff
@@ -36,6 +36,12 @@ const (
 	// ElasticQuotaValidatingWebhook enables validating webhook for ElasticQuotas creations or updates
 	ElasticQuotaValidatingWebhook featuregate.Feature = "ElasticValidatingWebhook"
 
+	//NodeValidatingWebhook enables validating webhook for Node Creation or updates
+	NodeValidatingWebhook featuregate.Feature = "NodeValidatingWebhook"
```

Comment:
Why does the webhook reject the node create/update when the slo-config conflicts?

Reply:
How about recording events when the slo-config duplicates on a node instead? The duplication of the slo-config may not be a fatal error that can block the node creation.

Reply:
okay